### DISCUSSION TITLE
Issue applying to Laravel

### DISCUSSION THREAD

**Discussion by Scottzozer** [TailAdmin/tailadmin-free-tailwind-dashboard-template#5] [General]
Hi All, 

Trying to use this template on a Laravel 10 project but keep running into issues when trying to merge it into my project. Right now I am getting an error on almost all of the custom CSS @apply classes saying they do not exist. Probably a noob mistake somewhere but I have added all the necessary libraries from the package.json, npm installed to generate all the node modules, added all the files, images, css, js to my project. Updated paths so they are in Laravel syntax but still getting these errors:

`style.css:40:3: The 'border-primary' class does not exist. If 'border-primary' is a custom class, make sure it is defined within a '@layer' directive.`

Anyone have any ideas on how to solve?

**Comment by dc-jahanzaib** [comment]
also multi select is not functional

**Reply by Coderamrin** [reply]
@dc-jahanzaib 
Can you share a screenshot?

**Comment by Aghiad90** [comment]
i have integrated this template with Laravel with no problem. did you include the style.css in your build?
